lib/vanagon/optparse.rb

```diff
@@ -2,44 +2,27 @@
 
 class Vanagon
   class OptParse
+
+    FLAGS = {
+        :workdir => ['-w DIR', '--workdir DIR', "Working directory where build source should be put (defaults to a tmpdir)"],
+        :configdir => ['-c', '--configdir DIR', 'Configs dir (defaults to $pwd/configs)'],
+        :target => ['-t HOST', '--target HOST', 'Configure a target machine for build and packaging (defaults to grabbing one from the pooler)'],
+        :engine => ['-e ENGINE', '--engine ENGINE', "A custom engine to use (defaults to the pooler) [base, local, docker, pooler currently supported]"],
+        :skipcheck => ['--skipcheck', 'Ship the `check` stage when building components'],
```

Comment:
Disregard what I said earlier about `skip-check` because we could probably make it work here with less effort than I thought? MMMMMMMM, nitpicky goodness.